app/models/chargeback.rb

```diff
@@ -182,7 +182,10 @@ def self.report_cb_model(model)
     model.gsub(/^Chargeback/, "")
   end
 
+  # @return true if this class name is a subclass of chargeback?
+  # looking up the subclasses takes too long (50ms)
+  # so checking that the class name starts with "Chargeback" (10 is length of that string)
   def self.db_is_chargeback?(db)
-    subclasses.collect(&:to_s).include?(db)
+    db && db.length > 10 && db.starts_with?("Chargeback")
```

Comment:
This works until we rename Chargeback, enable non Chargeback classes to do this....
This is fine for now, but I'm not sure what this feature is for... why do we want to check if this a chargeback subclass?
Is it expensive to have a constant of safe "db" class names and call safe_constantize("db").supports_some_feature?